app/controllers/albums.js

```diff
@@ -36,3 +36,12 @@ exports.buyAlbum = async (req, res, next) => {
     next(err);
   }
 };
+
+exports.listPurchasedAlbums = (req, res, next) =>
+  Album.getAlbumBy({
+    userId: req.params.userId
+  })
+    .then(albumUser => {
+      res.status(200).send(albumUser);
```

Comment:
Something very important is missing, you should call API for get the Data of every album